monkey/infection_monkey/network/relay/tcp.py

```diff
@@ -0,0 +1,54 @@
+import select
+from logging import getLogger
+from threading import Thread
+from typing import Callable
+
+from infection_monkey.transport.base import update_last_serve_time
+
+READ_BUFFER_SIZE = 8192
+SOCKET_READ_TIMEOUT = 10
+
+logger = getLogger(__name__)
+
+
+class SocketsPipe(Thread):
```

Comment:
Let's rename `tcp.py` to be `sockets_pipe.py` so the file name matches the class name.

Reply:
I was thinking the same thing